docs/spark-standalone.md

```diff
@@ -374,6 +374,24 @@ To run an interactive Spark shell against the cluster, run the following command
 
 You can also pass an option `--total-executor-cores <numCores>` to control the number of cores that spark-shell uses on the cluster.
 
+#Spark Properties
+
+Spark applications supports the following configuration properties specific to standalone Mode: 
+<table class="table">
```

Comment:
Nit: Mode -> mode
and newline before the table.

Reply:
Updated in the latest commit.